docs/tutorials/anomaly-detection.rst

```diff
@@ -0,0 +1,614 @@
+* **Slack App and Token**: the data pipeline output is a notifications to a Slack channel, check out the needed steps to `set up a Slack app and retrieve the Slack authentication token <https://github.com/aiven/slack-connector-for-apache-flink>`_.
+* `psql <https://www.postgresql.org/docs/current/app-psql.html>`_ a terminal based tool to interact with PostgreSQL where the threshold data will be stored.
+
+Create the Aiven services
+----------------------------
+
+In this section you'll create all the services needed to define the anomaly detection system via the `Aiven Console <https://console.aiven.io/>`_:
+
+* An :doc:`Aiven for Apache Kafka®</docs/products/kafka>` named ``demo-kafka`` for data streaming, this is where the stream of IoT sensor readings will land
+* An :doc:`Aiven for Apache Flink®</docs/products/flink>` named ``demo-flink`` for streaming data transformation, to define the anomaly detection queries
+* An :doc:`Aiven for PostgreSQL®</docs/products/postgresql>` named ``demo-postgresql`` for alerting thresholds storage and query
+
+
+Create an Aiven for Apache Kafka® service
+'''''''''''''''''''''''''''''''''''''''''''''
+
+The :doc:`Aiven for Apache Kafka </docs/products/kafka>` service is responsible to receive the inbound stream of IoT sensor readings. You can create the service with the following steps:
+
+1. Log in to the `Aiven web console <https://console.aiven.io/>`_.
+2. On the *Services* page, click **Create a new service**.
+
+   This opens a new page with the available service options.
+
+   .. image:: /images/platform/concepts/console_create_service.png
+      :alt: Aiven Console view for creating a new service
+
+3. Select **Apache Kafka®**.
+
+4. Select the cloud provider and region that you want to run your service on.
+
+5. Select `business-4` as service plan.
+
+5. Enter ``demo-kafka`` as name for your service.
+
+6. Click **Create Service** under the summary on the right side of the console
+
+Customise the Aiven for Apache Kafka service
+''''''''''''''''''''''''''''''''''''''''''''
+
+Now that your service is created, you need to customise its functionality. In the **Overview** tab of your freshly created service, you'll see a bunch of toggles and properties. Change these two:
+
+1. **Kafka REST API (Karapace)** > **Enable**
+
+   .. Note::
+
+    The **Kafka REST API** allows you to manage and query Apache Kafka via REST APIs. You'll use it to inspect the data in Apache Kafka from the Aiven Console.
+
+2. **Advanced configuration** > **Add configuration option** > ``kafka.auto_create_topics_enable``, switch the setting on and then click **Save advanced configuration**
+
+   .. Note::
+
+    The ``kafka.auto_create_topics_enable`` setting allows you to create new Apache Kafka® topics on the fly while pushing a first record. It avoids needing to create a topic in advance. To read more about the setting, check the :doc:`dedicated documentation </docs/products/kafka/howto/create-topics-automatically>`.
```

Comment:
Because this is a tutorial for the sake of a story flow I'd remove the note and refrase this section explaining why we need to set this setting